reports/issue-count-cycle-time.ts

```diff
@@ -0,0 +1,83 @@
+import {ProjectData, IssueCard} from '../interfaces';
+import * as rptLib from '../project-reports-lib';
+const tablemark = require('tablemark')
+import * as os from 'os';
+
+export type CycleTimeData = { [key: string]: CycleTimeStageData }
+interface CycleTimeStageData {
+  title: string,
+  cycletime: number,
+  limit: number,
+  count: number,
+  flag: boolean
+}
+
+interface CycleTimeRow {
+    worktype: string,
+    labels: string,
+    count: number,
+    "Cycle Time in days": string,
+    limit: number
+    flag: string
+}
+
+export interface IssueCardCycleTime extends IssueCard {
+  cycletime: number
+}
+
+export function getDefaultConfiguration(): any {
+    return <any>{};
+}
+
+export function process(config: any, projData: ProjectData, drillIn: (identifier: string, title: string, cards: IssueCard[]) => void): any {
+  let cycleTimeData = <CycleTimeData>{};
+  for (let cardType of config["report-on"]) {
+      let stageData = <CycleTimeStageData>{};
+      let cards = projData.stages["Done"];
+      console.log(`Found [${cards.length}] cards in Done state`);
+      let cardsForType = rptLib.filterByLabel(cards, cardType.toLowerCase());
```

Comment:
the counts can offer drill-in like wip limits.  call the drillIn function passed with the IssueCard[] and the engine does it for free.  If someone says, wow, that's a high cycle time and they want to see what's causing it they can drill in.  the title is the heading of the drill in sub report and the identifier is something unique.  probably cycle-{cardType} etc.  see wipLimits